components/common/header/authNav.js

```diff
@@ -82,17 +82,19 @@ const AuthNav = ({ user, onLogout }) => {
             >
               {user.email.charAt(0)}
             </Box>
-            {isExpanded ? (
-              <ChevronUpIcon
-                marginLeft='.5rem'
-                size='1.5rem'
-              />
-            ) : (
-              <ChevronDownIcon
-                marginLeft='.5rem'
-                size='1.5rem'
-              />
-            )}
+            {isExpanded
```

Comment:
sorry abt this; I upgraded to standard@16 because the standard we were using didn't like how I was using the `object?.chain?.thing`. the updated standard is very picky about how ternaries are written.